apps/paymaster-proxy/package.json

```diff
@@ -10,16 +10,16 @@
     "start": "node dist/runProxyService.js",
     "lint": "eslint \"**/*.{ts,tsx}\" && pnpm prettier --check \"**/*.{ts,tsx}\"",
     "lint:fix": "eslint \"**/*.{ts,tsx}\" --fix --quiet && pnpm prettier \"**/*.{ts,tsx}\" --write --loglevel=warn",
-    "generate-example-env": "ts-node scripts/generateExampleEnv.ts",
     "loadtest": "pnpm dlx artillery run loadtest/basic-loadtest.yml",
-    "docker:services:up": "docker compose up -d redis postgres",
-    "docker:up": "docker compose up -d redis postgres paymaster-proxy-app",
+    "docker:services:up": "nx run api-key-service:docker:up && docker compose up -d redis postgres",
+    "docker:up": "nx run api-key-service:docker:up && docker compose up -d redis postgres paymaster-proxy-app",
```

Comment:
not ideal but will keep doing this until we unify the monorepo into a single docker compose